client/lib/services/database.dart

```diff
@@ -57,6 +69,20 @@ Future<void> deleteCourse(String uid, String courseId) async {
   await courses.doc(courseId).delete();
 }
 
+DocumentReference<Map<String, dynamic>> newLectureRef(String courseID) {
+  return courses.doc(courseID).collection('audios').doc();
+}

```

Comment:
reorganized/reordered the functions to group them by level of concern and match the apps flow better but that screws up git tracking, the only new change is this function

Reply:
love when that happens, not a major issue so that's fine